install/first_google_boot.sh

```diff
@@ -207,6 +207,10 @@ function extract_spinnaker_kube_credentials() {
   local kube_project=$(get_instance_metadata_attribute "kube_project")
   local kube_zone=$(get_instance_metadata_attribute "kube_zone")
 
+  clear_instance_metadata "kube_cluster"
```

Comment:
That's a good point, I can fix that. I originally took the stance of "immutable infrastructure, re-provision" on error but this makes it a little easier to use.